pkg/diff/_future/README.md

```
@@ -0,0 +1,75 @@
+## Recipe-vs-Snapshot Drift Detection (deferred)
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)